app/workers/teacher_training_public_api/sync_all_providers_and_courses_worker.rb

```diff
@@ -4,6 +4,7 @@ class SyncAllProvidersAndCoursesWorker
     sidekiq_options retry: 3, queue: :low_priority
 
     def perform
+      SyncSubjects.perform_async
```

Comment:
I think this should no longer be async and we should change it to a normal service as such:
```
SyncSubjects.call
```
to allow this to be run sequentially before syncing providers and courses

Reply:
or maybe just call `.perform` actually

Reply:
Addressed